protocol-units/settlement/mcr/client/src/eth_client.rs

```diff
@@ -24,13 +20,17 @@ use alloy_signer_wallet::LocalWallet;
 use alloy_sol_types::sol;
 use alloy_transport::BoxTransport;
 use alloy_transport_ws::WsConnect;
-
 use anyhow::Context;
+use mcr_settlement_config::Config;
+use movement_types::BlockCommitment;
+use movement_types::{Commitment, Id};
+use serde_json::Value as JsonValue;
+use std::array::TryFromSliceError;
+use std::fs;
+use std::path::Path;
```

Comment:
And this is why I don't like the ungrouped imports, everything is mashed together. Ah well.

Reply:
I don't agree here. They are sorted and you can find one easily. If you separate them like in this example. Alloy is separated. Automatic import when they are missing (my IDE add them automatically) will be added to the alloy section which result a real mash up where you have two sorted collections of import and need to search in both. And if you add more separate collection, in the end everything is mashed. Otherwise you've to very each time you import that the new import is in the right place that remove all the interest of automatic import.

Reply:
OK, this seems less of a headache with automatically added imports.

Reply:
I also prefer to group imports a bit.